package.json

```diff
@@ -19,6 +19,7 @@
     "Libraries",
     "ReactWindows/ReactNative",
     "ReactWindows/ReactNative.Shared",
+     "ReactWindows/ReactNative.Net46",
     "ReactWindows/ChakraBridge",
```

Comment:
This isn't supposed to go in the package.json yet.  Or is this just a temporary step to unblock until you create the build process for separate packages?